dirty_memory_manager.hh

```diff
@@ -144,23 +144,26 @@ public:
     bool empty() const noexcept { return _blocked_requests.empty(); }
 };
 
-class region_group;
-class memory_hard_limit;
+// Groups regions for the purpose of statistics.  Can be nested.
```

Comment:
Is nesting still possible? I thought this merging eliminates the only use-case for it.

Reply:
They can no longer be nested.